.github/workflows/tox.yaml

```diff
@@ -25,12 +25,12 @@ jobs:
       - name: Install Tox and any other packages
         run: pip install tox
       - name: Run Tox
-        run: tox -e py -e docs # Run tox using the version of Python in `PATH`
+        run: tox -e py -e torch -e tensorFlow -e docs # Run tox using the version of Python in `PATH`
```

Comment:
this could be shortened to
```
tox -e py,torch,tensorflow,docs
```
I just found out about this syntax :)

Reply:
Done

Reply:
Actually you have to write
tox -e py, torch, tensorflow, -e docs

as otherwise docs will not be build.